File: server/internal/entity/office.go
package entity

import "github.com/gofrs/uuid/v5"

type Office struct {
	ID                  uuid.UUID       `json:"id"`
	SalePointName       string          `json:"sale_point_name"`
	Address             string          `json:"address"`
	Status              string          `json:"status"`
	OpenHours           []OpenHours     `json:"open_hours"`
	RKO                 *bool           `json:"rko"`
	OpenHoursIndividual []OpenHours     `json:"open_hours_individual"`
	Type                string          `json:"type"`
	SalePointFormat     string          `json:"sale_point_format"`
	SuoAvailability     *bool           `json:"suo_availability"`
	HasRamp             *bool           `json:"has_ramp"`
	Position            *Position       `json:"position"`
	Distance            int             `json:"distance"`
	KEP                 *bool           `json:"kep"`
	MyBranch            bool            `json:"my_branch"`
	Services            []OfficeService `json:"services"`
}

File: transformer/db/models/office.py
from peewee import *
from .position import Position
from .base_model import BaseModel

import uuid


class Office(BaseModel):
  id = UUIDField(primary_key=True, default=uuid.uuid4)
  sale_point_name = CharField(max_length=255)
  address = CharField(max_length=400)
  status = CharField(max_length=255)
  rko = BooleanField(null=True)
  type = CharField(max_length=255)
  sale_point_format = CharField(max_length=255)
  suo_availability = BooleanField(null=True)
  has_ramp = BooleanField(null=True)
  distance = IntegerField()
  kep = BooleanField(null=True)
  my_branch = BooleanField()
  position = ForeignKeyField(Position, related_name='position')
